standalone-metastore/metastore-server/src/main/sql/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql

```diff
@@ -160,20 +160,20 @@ CREATE OR REPLACE VIEW `COMPACTIONS`
   `C_TBLPROPERTIES`
 ) AS
 SELECT
```

Comment:
The original DDL was invalid. I copied [the non-upgrade DDL](https://github.com/apache/hive/blob/74b9c88aced9407351f6635769a4bd48214fca1e/standalone-metastore/metastore-server/src/main/sql/hive/hive-schema-4.0.0-alpha-2.hive.sql#L1188-L1274).

```
Error: Error while compiling statement: FAILED: SemanticException Schema of both sides of union should match. (state=42000,code=40000)
2024-06-27T02:42:12,201 ERROR [main] schematool.MetastoreSchemaTool: Upgrade FAILED! Metastore state would be inconsistent !!
Upgrade FAILED! Metastore state would be inconsistent !!
2024-06-27T02:42:12,201 ERROR [main] schematool.MetastoreSchemaTool: Underlying cause: java.io.IOException : Schema script failed, errorcode 2
Underlying cause: java.io.IOException : Schema script failed, errorcode 2
2024-06-27T02:42:12,201 ERROR [main] schematool.MetastoreSchemaTool: Use --verbose for detailed stacktrace.
Use --verbose for detailed stacktrace.
2024-06-27T02:42:12,201 ERROR [main] schematool.MetastoreSchemaTool: *** schemaTool failed ***
*** schemaTool failed ***
```